README

```diff
@@ -0,0 +1,16 @@
+================================================================================
+  softdevbib
+================================================================================
+
+This repository contains a curated .bib file of BibTeX entries for the research
+field(s) of interest to the Software Development Team <http://soft-dev.org>.
+There is by definition no original work in this, so the .bib file is in
+the public domain.
+
+To help us maintain a high-quality .bib file, this repository also includes an
+(optional) utility called prebib. This has two purposes: as a linter for .bib
+files (i.e. to spot obvious errors and/or common style mistakes); and as a way
+of excluding entries with the -x switch. For example, if you need a bibliography
+which excludes months execute "prebib -x month softdevbib.bib > bib.bib". Please
```

Comment:
Looks like prebib already excludes some fields without specifying `-x`. You will need to mention this.